torch/onnx/__init__.py

```diff
@@ -103,11 +103,17 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
         export_params (bool, default True): if True, all parameters will
             be exported. Set this to False if you want to export an untrained model.
             In this case, the exported model will first take all of its parameters
-            as arguments, with the ordering as specified by ``model.state_dict().values()``
+            as arguments, with the ordering as specified by ``model.state_dict().values()``.
+            This helps on stripping parameters from the model so that large models could be export
+            succssfully. Besides this, if this is False, no matter the model will be export in
+            inference mode or training mode, the optimization which may adjust graph inputs will
+            be skipped. For example, Conv and BatchNorm fusion.
         verbose (bool, default False): if True, prints a description of the
             model being exported to stdout.
         training (enum, default TrainingMode.EVAL):
-            * ``TrainingMode.EVAL``: export the model in inference mode.
+            * ``TrainingMode.EVAL``: export the model in inference mode. The value of parameter
```

Comment:
I don't think this sentence is necessary for training mode comments.
If you want to keep it, consider re-writing the comment and make it more clear.

Reply:
For TrainingMode.TRAINING value, its comments mentioned that this mode will disable optimizations. So I think we also need to mention that some optimizations will also be disabled even for TrainingMode.EVAL value.

Agree to update the content to make it more clear.